rrrspec-client/lib/rrrspec/client/slave_runner.rb

```diff
@@ -3,6 +3,9 @@
 require 'timeout'
 require 'rspec/core/formatters'
 
+include Cucumber::RbSupport::RbDsl
```

Comment:
Sure, removed the cucumber dependency add added support for adding runners as plugins. 
You can see the cucumber runner plugin at https://github.com/Seedrs/rrrspec-cucumber2